pkg/chains/chains.go

```diff
@@ -91,6 +91,18 @@ var (
 		CctxGateway: CCTXGateway_observers,
 	}
 
+	// SolanaMainnet is Solana mainnet
+	SolanaMainnet = Chain{
+		ChainName:   ChainName_solana_mainnet,
+		ChainId:     900,
```

Comment:
From which official source did we see this mainnet chain ID? An unofficial place I saw is: https://docs.expand.network/important-ids/chain-id

Reply:
Yeah, it's unofficial. Since the chain ID is used in the signature unlike Bitcoin, we might want to check carefully

Reply:
Created issue to define final values here: https://github.com/zeta-chain/node/issues/2421

Reply:
a843cc4cc